nncf/definitions.py

```diff
@@ -14,3 +14,9 @@
 import os
 NNCF_PACKAGE_ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
 HW_CONFIG_RELATIVE_DIR = "common/hardware/configs"
+
+# Environment variables below, if set, mark the execution environment
+# so that certain actions within NNCF proper, such as telemetry event collection or
+# debug dumps, are performed or not performed
+NNCF_CI_ENV_VAR_NAME = "NNCF_CI"  # Must be set in CI environments
+NNCF_DEV_ENV_VAR_NAME = "NNCF_DEV" # Must be set in environments of the NNCF dev team machines
```

Comment:
AFAIK, the analytics itself should filter out intel IPs. But if we are going to go with these variables it would be better to handle this in the setup.py (like set it up in the Python env when `python setup.py develop` is used).

Reply:
These variables are for internal usage only. If someone installs NNCF in the editable mode (i.e. `python setup.py develop` or `pip install -e .`, this does not necessarily mean that this is an NNCF team member - that might be a user who just likes to install their packages in editable mode.